readme.md

```diff
@@ -53,11 +53,13 @@ So a lot of you may have lots of open windows at once.
    If you want to be super fast and efficient "You need to learn how to use Spectacle".
    Once you learn some of the shortcuts you will just need to use your "Muscle Memory" to use them.
 
+_Note:_ Spectacle is only for Mac in alternative for windows you have AquaSnap that works the same way.
+
 # Spectacle Installation
 
 First of all access the website :https://www.spectacleapp.com/
 
-![Spectacle](https://i.postimg.cc/28frXpZL/9688-AE1-C-A633-427-D-91-EE-56-CABA0-DF2-D6-1-105-c.jpg)
+![Spectacle](https://cdn1.imggmi.com/uploads/2020/1/7/00355c30da035c3f844a03854784892c-full.png)
```

Comment:
I can't see the new image on my side. Please confirm you can see it:
https://github.com/dwyl/dev-setup/blob/6a2785f5bb95839502f40cd539d17355177f7e9d/readme.md#spectacle-installation
![image](https://user-images.githubusercontent.com/194400/71909252-89b5b180-3167-11ea-9b24-189a5456f704.png)

Reply:
@nelsonic Sorry I didn't see that the image was like that

Reply:
@miguelmartins17 remember that you can drag-and-drop images (_or paste them_) into any GitHub Comment and not have to use another image service.

Reply:
@nelsonic I didn't know that I could drag images. So I was using an upload program.